src/main/java/com/hcl/appscan/jenkins/plugin/auth/ASoCCredentials.java

```diff
@@ -73,5 +86,12 @@ public FormValidation doCheckUsername(@QueryParameter String username) {
 		public FormValidation doCheckPassword(@QueryParameter String password) {
 			return FormValidation.validateRequired(password);
 		}
+
+        public FormValidation doCheckCertificates(@QueryParameter Boolean certificates,@QueryParameter String url){
```

Comment:
If I'm not mistaken, this will display an error anytime the server field is not empty and it contains the string "appscan.com". I don't think that's what we want here. It would be better if we did one of two things:

1. Disable the checkbox if the field is empty or contains "appscan.com".
OR
2. Have this check only display an error if the checkbox is selected, and the field value contains "appscan.com".

Reply:
Make the required changes, the validation error will be shown only when the checkbox is selected.